contracts/hardhat.config.cjs

```diff
@@ -129,4 +129,29 @@ task('read-data-feed', 'Read latest data from DataFeedConsumerMock')
     }
   })
 
+task('set-burn-fee-ratio', 'Change the burn fee ratio')
+  .addParam('ratio', 'New burn fee ration')
+  .addParam('address', 'Prepayment contract address')
+  .setAction(async (taskArgs, hre) => {
+    let _consumer
```

Comment:
I think we should update the variable names since they are confusing in this context. I understand that this was taken from `read-data-feed` tasks which is operated by a `consumer`, but in this case we need account of `owner`. In the context of this repository, we call that signer a `deployer`.

Reply:
@martinkersner That is really good point. Now I changed the `consumer` to `deployer`. Please, check it out.